content/about/about_people.rst

```diff
@@ -316,3 +316,41 @@ oweng
 .. raw:: html
 
   <br/>
+
+`Ayden Murphy-Kirsch`_
```

Comment:
Can you please move your section to just below Ethan? That should be alphabetical for the students.

Reply:
Done